core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -90,6 +92,9 @@
   /** The master registry. */
   private final MasterRegistry mRegistry;
 
+  /** The block master. */
+  private final BlockMaster mBlockMaster;
```

Comment:
I don't think we need to save this. We can just use the master registry to get the block master.